### PR TITLE
feat(feedback): toggle option for relay to produce to ingest-feedback

### PR DESCRIPTION
**Pull request by aliu3ntry**
Follow-up to (merge after) https://github.com/getsentry/self-hosted/pull/3060

### CODE REVIEW

File: sentry/sentry.conf.example.py [hunk 1]
```diff
@@ -134,6 +134,9 @@ def get_internal_network():
 
 KAFKA_CLUSTERS["default"] = DEFAULT_KAFKA_OPTIONS
 
+# Temporary flag to test a new ingest consumer for user feedback.
+SENTRY_OPTIONS["feedback.ingest-topic.rollout-rate"] = 1.0
```
Comment (hubertdeng123) [inline sentry/sentry.conf.example.py:138]: Will the default value of this option be changed in sentry? How will we know it is time to remove this temporary flag?

Comment (aliu3ntry) [reply sentry/sentry.conf.example.py:138]: The default value is 0.0 and will not be changed

Comment (aliu3ntry) [reply sentry/sentry.conf.example.py:138]: As soon as this ingest topic is tested and confirmed to work in self-hosted, I will remove references to the flag in all repos.

Comment (hubertdeng123) [reply sentry/sentry.conf.example.py:138]: Got it, so essentially the option will not be needed in the future after the references to the flag is removed

Comment (aliu3ntry) [reply sentry/sentry.conf.example.py:138]: Yup!

**Comment by aliu3ntry** [comment]
circling back later, see comment on https://github.com/getsentry/self-hosted/pull/3060

**Pull request closed without merging**